addons/web/static/src/legacy/scss/utils.scss

```diff
@@ -221,10 +221,14 @@
     @return $-with-support-font;
 }
 
-// Function to remove all null values of a map
+// Function to remove all null values of a map. Also transforms `'True'` values
+// into a boolean `true`.
```

Comment:
I'd say let's be a bit more consistent and handle "False" too (even if it was not part of the bug here). Also add a slight explanation about why this is done in this function. You can also add this to justify it more:
```scss
// TODO in master, this should be moved in website, as the only use case is in website user values files
```